sway/commands/client.c

```diff
@@ -0,0 +1,79 @@
+#include "log.h"
+#include "sway/commands.h"
+#include "sway/config.h"
+#include "sway/tree/container.h"
+
+static bool parse_color(char *hexstring, float (*dest)[4]) {
```

Comment:
No need for a pointer here. Also, add `static` so that the min number of elements is 4:

```c
float dest[static 4]
```